src/app/constants/constants.ts

```diff
@@ -1,9 +1,9 @@
 export default Object.freeze({
 	APP_VERSION: process.env.npm_package_version,
-	APP_NAME: 'app-name',
-	DB_NAME_DEGEN: 'db-name',
+	APP_NAME: 'Governator',
+	// DB_NAME_DEGEN: 'db-name',
 
-	MONGODB_URI_PARTIAL: `${process.env.MONGODB_PREFIX}://${process.env.MONGODB_USERNAME}:${process.env.MONGODB_PASS}@${process.env.MONGODB_CLUSTER}/`,
+	// MONGODB_URI_PARTIAL: `${process.env.MONGODB_PREFIX}://${process.env.MONGODB_USERNAME}:${process.env.MONGODB_PASS}@${process.env.MONGODB_CLUSTER}/`,
```

Comment:
Commented out?

Reply:
Yes, can be deleted actually since bot is not connecting directly to db. I have commented out for now, still need a more thorough sweep to remove all references to db